### ISSUE TITLE
ParseUser.signUpInBackground, ParseUser.loginInBackground and ParseUser.logOut are not implemented

### ISSUE THREAD

**Issue by lxknvlk** [thiagolocatelli/parse4j#88]
This is what there is:

    public void signUpInBackground(SignUpCallback callback) {
    }

    public static void loginInBackground(String username, String password, LoginCallback callback) {
    }

empty methods.

**Comment by SeloSlav** [comment]
Yeah I encountered this while building an app, not realizing they were not implemented. You can't currently store sessions on a Java web app (like Springboot) using Parse (and I don't think the devs here have or will be working on it anytime soon). You may have to use the Javascript SDK instead of Java unless you build these methods yourself, unfortunately. I wouldn't know how at the moment, nor have the time.
